test_scripts/API/VehicleData/common.lua

```diff
@@ -406,25 +466,65 @@ end
 --! @return: none
 --]]
 function m.processSubscriptionRPC(pRPC, pParam, pAppId, isRequestOnHMIExpected)
```

Comment:
Maybe add 'p' to name parameter "isRequestOnHMIExpected"?
```suggestion
function m.processSubscriptionRPC(pRPC, pParam, pAppId, pIsRequestOnHMIExpected)
```

Reply:
Propose to leave this as is since the same name is used in other existing functions